src/test/java/org/drugis/trialverse/dataset/repository/DatasetReadRepositoryTest.java

```diff
@@ -100,10 +98,10 @@ public void testQueryDatasets() {
     Account account = new Account(1, "username", "firstName", "lastName", "foo@bar.com");
     String datasetLocation = "loc1";
     String versionKey = "version1";
-    VersionMapping versionMapping = new VersionMapping(1, datasetLocation, account.getEmail(), versionKey);
+    VersionMapping versionMapping = new VersionMapping(1, datasetLocation, account.getEmail(), versionKey, false, null);
```

Comment:
these null arguments can be prevented with extra constructor for default 'null' archived state I think

Reply:
The new constructor would only be used in the tests. Can live with the current implementation?